backend/src/org/commcare/suite/model/View.java

```diff
@@ -0,0 +1,29 @@
+package org.commcare.suite.model;
+
+import org.javarosa.core.model.instance.DataInstance;
+
+import java.util.Hashtable;
+import java.util.Vector;
+
+/**
+ * An entry action used for viewing data; main difference from an Entry is that
+ * it doesn't end in launching form entry.
+ *
+ * @author Phillip Mates (pmates@dimagi.com).
+ */
+public class View extends EntryBase {
```

Comment:
Can this have a more descriptive name? Feels weird to name something "View" when it is something much more specific than anything like the notion of Android's View class